kahypar/partition/recursive_bisection.h

```diff
@@ -233,6 +234,12 @@ static inline void partition(Hypergraph& input_hypergraph,
       HypergraphPtr(hg_without_fixed_vertices.first.release(),
                     delete_hypergraph);
     fixed_vertex_free_to_input = hg_without_fixed_vertices.second;
+
+    for ( PartitionID block = 0; block < original_context.partition.k; ++block ) {
+      rb_context.partition.max_part_weights[block] -= input_hypergraph.fixedVertexPartWeight(block);
+    }
+    rb_context.partition.use_individual_part_weights = true;
```

Comment:
This should also not be needed since we copy-construct the `rb_context` from the `original_context`, right?

Reply:
Should be answered with my previous comment. Even if we do not use individual block weights, removing fixed vertices leads to individual block weights for initial Partitioning.